content/developers/overview/about-githubs-apis.md

```diff
@@ -9,11 +9,9 @@ versions:
   github-ae: '*'
 ---
 
-There are two stable versions of the GitHub API: the [REST API](/v3/) and the [GraphQL API](/v4/).
-
-When using the REST API, we encourage you to [request v3 via the `Accept` header](/v3/media/#request-specific-version).
-
-For information on using the GraphQL API, see the [v4 docs](/v4/).
+You can use the stable GitHub API via:
+- [GraphQL API](https://docs.github.com/en/free-pro-team@latest/v4)
+- [REST API](https://docs.github.com/en/free-pro-team@latest/v3) where we encourage you to [request v3 via the `Accept` header.](https://docs.github.com/en/free-pro-team@latest/v3/media/#request-specific-version)
```

Comment:
Let's keep the sentences as is, but format them as one paragraph instead of three.

```suggestion
There are two stable versions of the GitHub API: the [REST API](/v3/) and the [GraphQL API](/v4/). When using the REST API, we encourage you to [request v3 via the `Accept` header](/v3/media/#request-specific-version). For information on using the GraphQL API, see the [v4 docs](/v4/).
```